README.md

```diff
@@ -559,6 +560,249 @@ The model can also be downloaded and served locally.
 
+### 🧪 Evaluate the newly trained model
+
+You can use the `ilab` command to evaluate the models you are training with several benchmarks. Currently, four benchmarks are supported.
+
+| Benchmark | Measures | Full Name | Description |
+| --- | --- | --- | --- |
+| MMLU | Knowledge | Massive Multitask Language Understanding | TODO |
+| MMLUBranch | Knowledge | N/A | TODO |
+| MTBench | Skills | Multi-turn Benchmark | TODO |
+| MTBenchBranch | Skills | N/A | TODO |
+
+> [!NOTE]
+> MTBench and MTBenchBranch use [prometheus-8x7b-v2.0](https://huggingface.co/prometheus-eval/prometheus-8x7b-v2.0) as the judge model by default. While you do not **need** to use this model as your judge, it is strongly recommended. You can download it via `ilab model download`.
+
+#### Running MMLU
+
+Below is an example of running MMLU on a local model with minimal tasks:
+
+```bash
+$ export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true   # don't set this if you want to run full MMLU 
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ ilab model evaluate --benchmark mmlu --model ILAB_MODELS_DIR/instructlab/granite-7b-lab
+...
+# KNOWLEDGE EVALUATION REPORT
+
+## MODEL
+/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+### AVERAGE:
+0.45 (across 3)
+
+### SCORES:
+mmlu_abstract_algebra - 0.35
+mmlu_anatomy - 0.44
+mmlu_astronomy - 0.55
+```
+
+Below is an example of running MMLU on a Hugging Face model with minimal tasks:
+
+```bash
+$ export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true   # don't set this if you want to run full MMLU 
+$ ilab model evaluate --benchmark mmlu --model instructlab/granite-7b-lab
+...
+# KNOWLEDGE EVALUATION REPORT
+
+## MODEL
+instructlab/granite-7b-lab
+
+### AVERAGE:
+0.45 (across 3)
+
+### SCORES:
+mmlu_abstract_algebra - 0.35
+mmlu_anatomy - 0.44
+mmlu_astronomy - 0.55
+```
+
+> [!NOTE]
+> Currently, MMLU can only be run against against safetensors, either locally or on Hugging Face. GGUFs are not currently supported.
+
+#### Running MMLUBranch
+
+Below is an example of running MMLUBranch with local safetensor models:
+
+```bash
+$ export ILAB_TAXONOMY_DIR=/home/example-user/.local/share/instructlab/taxonomy
+$ export ILAB_TASKS_DIR=/home/example-user/.local/share/instructlab/taxonomy
+$ ilab model evaluate --benchmark mmlu_branch --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab --tasks-dir $ILAB_TASKS_DIR
+...
+# KNOWLEDGE EVALUATION REPORT
+
+## BASE MODEL
+/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+## MODEL
+/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+### AVERAGE:
++0.0 (across 1)
+
+### NO CHANGE:
+1. tonsils
+```
+
+Below is an example of running MMLUBranch with Hugging Face models:
+
+```bash
+$ export ILAB_TAXONOMY_DIR=/home/example/.local/share/instructlab/taxonomy
+$ export ILAB_TASKS_DIR=/home/example-user/.local/share/instructlab/taxonomy
```

Comment:
ditto; why both? Also, replace `$HOME`. (There are more cases of the same; all should be fixed.)